resources/js/contest-judge-results/vote.tsx

```diff
@@ -39,17 +47,17 @@ export default function Vote(props: Props) {
         {props.vote.scores.map((score) => {
           const category = props.contest.scoring_categories.find((x) => x.id === score.contest_scoring_category_id);
 
-          return (
+          return category != null && (
             <div key={score.id} className='contest-judge-results-scores__row'>
               <div className='contest-judge-results-scores__col'>
-                <div className='contest-judge-entry__description-icon' title={category?.description}>
+                <div className='contest-judge-entry__description-icon' title={category.description}>
```

Comment:
unrelated but the class doesn't really do anything useful apart of being a random element of an entirely different block